target/linux/armsr/armv7/config-6.1

```diff
@@ -42,36 +55,188 @@ CONFIG_CPU_SPECTRE=y
+CONFIG_CRYPTO_DEV_STM32_CRC=y
+CONFIG_CRYPTO_DEV_STM32_CRYP=y
+CONFIG_CRYPTO_DEV_STM32_HASH=y
+CONFIG_CRYPTO_SHA1=y
 CONFIG_DEBUG_LL_INCLUDE="mach/debug-macro.S"
 CONFIG_DMA_OPS=y
+CONFIG_DMA_VIRTUAL_CHANNELS=y
+CONFIG_DMA_SHARED_BUFFER=y
+CONFIG_DRM_STM=y
+CONFIG_DRM_STM_DSI=y
 CONFIG_EDAC_ATOMIC_SCRUB=y
+# CONFIG_FTM_QUADDEC is not set
 CONFIG_GENERIC_IRQ_MULTI_HANDLER=y
 CONFIG_GENERIC_VDSO_32=y
+CONFIG_GENERIC_IRQ_CHIP=y
+CONFIG_GENERIC_PHY=y
+CONFIG_GENERIC_PINCONF=y
+CONFIG_GENERIC_PINCTRL_GROUPS=y
+CONFIG_GENERIC_PINMUX_FUNCTIONS=y
+# CONFIG_KEYBOARD_ST_KEYSCAN is not set
+CONFIG_GPIO_GENERIC=y
+CONFIG_GPIO_GENERIC_PLATFORM=y
+CONFIG_GPIO_SYSCON=y
+CONFIG_GPIO_SYSFS=y
 CONFIG_HARDEN_BRANCH_PREDICTOR=y
 CONFIG_HAVE_SMP=y
+CONFIG_HW_RANDOM_OPTEE=y
+CONFIG_HW_RANDOM_STM32=y
+CONFIG_HWSPINLOCK_STM32=y
 CONFIG_HZ_FIXED=0
-CONFIG_HZ_PERIODIC=y
+CONFIG_I2C=y
+CONFIG_I2C_BOARDINFO=y
+CONFIG_I2C_SLAVE=y
+CONFIG_I2C_SMBUS=y
+CONFIG_I2C_STM32F4=y
+CONFIG_I2C_CHARDEV=y
+CONFIG_I2C_DEMUX_PINCTRL=y
+CONFIG_I2C_MUX=y
+CONFIG_I2C_MUX_PCA954x=y
+CONFIG_I2C_MUX_PINCTRL=y
+CONFIG_I2C_STM32F7=y
+CONFIG_IIO_STM32_LPTIMER_TRIGGER=y
+# CONFIG_INTEL_QEP is not set
+# CONFIG_INTERRUPT_CNT is not set
+# CONFIG_MICROCHIP_TCB_CAPTURE is not set
+CONFIG_JFFS2_FS=y
+CONFIG_MACH_STM32MP13=y
+CONFIG_MACH_STM32MP157=y
+CONFIG_MFD_CORE=y
+# CONFIG_MFD_STM32_TIMERS is not set
+# CONFIG_MTD_ST_SPI_FSM is not set
+CONFIG_MFD_STM32_LPTIMER=y
 CONFIG_MIGHT_HAVE_CACHE_L2X0=y
+CONFIG_MMC=y
+CONFIG_MMC_ARMMMCI=y
+CONFIG_MMC_CQHCI=y
+CONFIG_MMC_RICOH_MMC=y
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_STM32_SDMMC=y
 CONFIG_MODULES_USE_ELF_REL=y
+CONFIG_MTD_BLOCK=y
 CONFIG_NEON=y
 CONFIG_NR_CPUS=4
+CONFIG_NVMEM_STM32_BSEC_OPTEE_TA=y
+CONFIG_NVMEM_STM32_ROMEM=y
+CONFIG_OF_DYNAMIC=y
 CONFIG_OLD_SIGACTION=y
 CONFIG_OLD_SIGSUSPEND3=y
+CONFIG_OPTEE=y
 CONFIG_OUTER_CACHE=y
 CONFIG_OUTER_CACHE_SYNC=y
 CONFIG_PAGE_OFFSET=0xC0000000
+CONFIG_PCS_XPCS=y
 CONFIG_PERF_USE_VMALLOC=y
+CONFIG_PINCTRL_STM32=y
+# CONFIG_POWER_RESET_ST is not set
+CONFIG_PHYLINK=y
+CONFIG_PHY_STM32_USBPHYC=y
+# CONFIG_PHY_MIPHY28LP is not set
+CONFIG_PINCTRL_SINGLE=y
+CONFIG_POWER_RESET_GPIO=y
+CONFIG_POWER_RESET_GPIO_RESTART=y
+CONFIG_POWER_RESET_SYSCON=y
+CONFIG_POWER_RESET_SYSCON_POWEROFF=y
+CONFIG_PWM_STM32=y
+CONFIG_PWM_STM32_LP=y
+CONFIG_REGMAP_I2C=y
+CONFIG_REGMAP_SPI=y
+CONFIG_REGULATOR_GPIO=y
+CONFIG_REGULATOR_PWM=y
+CONFIG_REGULATOR_STM32_BOOSTER=m
+CONFIG_REGULATOR_STM32_PWR=y
+CONFIG_REGULATOR_STM32_VREFBUF=y
+CONFIG_RESET_CONTROLLER=y
+CONFIG_RFKILL_GPIO=y
+CONFIG_RPMSG=y
+# CONFIG_RPMSG_CHAR is not set
+# CONFIG_RPMSG_CTRL is not set
+CONFIG_RPMSG_NS=y
+CONFIG_RPMSG_TTY=y
+CONFIG_RPMSG_VIRTIO=y
+# CONFIG_RTC_DRV_ST_LPC is not set
+CONFIG_RTC_DRV_OPTEE=y
+CONFIG_RTC_DRV_STM32=y
+CONFIG_RTC_I2C_AND_SPI=y
 CONFIG_RTC_MC146818_LIB=y
-CONFIG_SERIAL_OF_PLATFORM=y
+CONFIG_SD_ADC_MODULATOR=y
+CONFIG_SENSORS_IIO_HWMON=y
+CONFIG_SERIAL_STM32=y
+CONFIG_SERIAL_STM32_CONSOLE=y
 CONFIG_SMP_ON_UP=y
+CONFIG_SND=y
```

Comment:
I would prefer if sound is in a kmod package

Reply:
I have removed the SND support, as it's not required for now. In the future, the already defined `kmod-sound-core` could be added as kmod package.